virtcontainers/virtiofsd.go

```diff
@@ -125,11 +150,23 @@ func (v *virtiofsd) args() ([]string, error) {
 	}
 
 	args := []string{
+		// Send logs to syslog
+		"--syslog",
+		// foreground operation
 		"-f",
+		// cache mode for virtiofsd
 		"-o", "cache=" + v.cache,
+		// disable posix locking in daemon: bunch of basic posix locks properties are broken
+		// apt-get update is broken if enabled
 		"-o", "no_posix_lock",
+		// shared directory tree
 		"-o", "source=" + v.sourcePath,
-		"-o", "vhost_user_socket=" + v.socketPath,
+		// fd number of vhost-user socket
```

Comment:
why vhost-user, isnt FdSocketNumber  just a unix socket?

Reply:
Yes it is both, it is unix socket for vhost user connection, I wanted to keep the same as virtiofsd help `virtiofsd --help`